Lib/test/test_sqlite3/test_factory.py

```diff
@@ -66,7 +66,15 @@ class Factory(sqlite.Connection):
             def __init__(self, *args, **kwargs):
                 super(Factory, self).__init__(*args, **kwargs)
 
-        con = sqlite.connect(":memory:", 5.0, 0, None, True, Factory)
+        regex = (
+            r"Passing more than 1 positional argument to _sqlite3.Connection\(\) "
+            r"is deprecated. Parameters 'timeout', 'detect_types', "
+            r"'isolation_level', 'check_same_thread', 'factory', "
+            r"'cached_statements' and 'uri' will become keyword-only "
+            r"parameters in Python 3.15."
+        )
+        with self.assertWarnsRegex(DeprecationWarning, regex):
+            con = sqlite.connect(":memory:", 5.0, 0, None, True, Factory)
```

Comment:
All warning tests should also check `filename` to ensure that the stacklevel argument is correct.